packages/liferay-js-insights/.yarnrc

```diff
@@ -0,0 +1,3 @@
+# Make `yarn version` produce the right commit message and tag for this package.
+version-tag-prefix "liferay-js-insights/v"
+version-git-message "chore: prepare lifeiray-js-insights/v%s"
```

Comment:
Please fix `lifeiray` to `liferay` before merging.

Reply:
🙈 I swear I did fix this one... probably lost the commit 🤔